src/commands/test/runner.py

```diff
@@ -159,10 +158,24 @@ async def empty(
         include_paths: Optional[List[str]] = None,
     ):
         env = cls(is_test_fail_enabled, include_paths or [])
-        env.starknet = await Starknet.empty()
+        env.starknet = await ForkableStarknet.empty()
         env.test_contract = await env.starknet.deploy(contract_def=test_contract)
         return env
 
+    def fork(self):
+        assert self.starknet
+        assert self.test_contract
+
+        n_env = TestExecutionEnvironment(
+            is_test_fail_enabled=self._is_test_fail_enabled,
```

Comment:
I think the `is_test_fail_enabled` is no longer used.

Reply:
True, merge conflicts